package.json

```diff
@@ -54,6 +54,11 @@
     "@angular/cli": "~11.0.7",
     "@angular/compiler-cli": "~11.0.9",
     "@angular/language-service": "~11.0.9",
+    "@docusaurus/core": "^2.0.0-beta.0",
+    "@docusaurus/module-type-aliases": "^2.0.0-beta.0",
+    "@docusaurus/preset-classic": "^2.0.0-beta.0",
+    "@docusaurus/theme-common": "^2.0.0-beta.0",
+    "@mdx-js/react": "^1.6.21",
```

Comment:
wdf 🤣

Reply:
The v2 beta is recommended. And yes, Docusaurus is built on top of React and MDX. It's developed by Facebook.